libs/clients/finance/src/lib/finance.ts

```diff
@@ -38,20 +43,30 @@ export class FinanceService extends RESTDataSource {
     request.headers.set('X-Road-Client', this.options.xroadClientId)
   }
 
+  handleError(error: string): any {
```

Comment:
We must log the exception, something like
Also should return undefined? at least we know it'll always throw
```suggestion
  handleError(error: string, exception: Error): any {
```